### PR TITLE
Use Rubocop plugin for CodeClimate.

### PR DESCRIPTION
**Pull request by simi**
CodeClimate has custom configuration. Currently we lint using Rubocop and code coverage is done using CI CodeCov action. We can give up on CodeClimate at all or try to follow rubocop rules (I'm tryin this in this PR).

### CODE REVIEW

**Review by jenshenny — COMMENTED** [review]
I like this change!

**Comment by simi** [comment]
Actually we can remove codeclimate at all if I understand it well now. Codecov is doing its job well and rubocop is being checked at CI.

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,17 +1,4 @@
...
-- "**/vendor/"
-- "**/*.d.ts"
-- "app/models/user.rb"
-method-count:
-    config:
-      threshold: 25
+plugins:
+  rubocop:
+    enabled: true
+    channel: "rubocop-1-23-0"
```
[the remark this answers is not in the document]
Comment (simi) [reply .codeclimate.yml:4]: some ancient one like 0.49 :-o

**Comment by deivid-rodriguez** [comment]
I used this for a while but eventually gave up, mainly due to https://github.com/codeclimate/codeclimate-rubocop/issues/93. If we can get rid of codeclimate, I would do it.

**Comment by simi** [comment]
let's remove CC

**Pull request closed without merging**